config.py

```
@@ -29,415 +21,270 @@
-__With VCPlayer you can easily record all your video chats.
-By default telegram allows you to record for a maximum duration of 4 hours. 
-An attempt to overcome this limit has been made by automatically restarting the recording after  4 hours__
-
-Command: **/record**
-
-AVAILABLE CONFIGURATIONS:
-1. Record Video: __If enabled both the video and audio of the stream will be recorded, otherwise only audio will be recorded.__
 
-2. Video dimension: __Choose between portrait and landscape dimensions for your recording__
+    RECORDER_HELP = """
+__ᴇᴀꜱɪʟʏ ʀᴇᴄᴏʀᴅ ᴀʟʟ ʏᴏᴜʀ ᴠɪᴅᴇᴏ ᴄʜᴀᴛꜱ ᴜꜱɪɴɢ ᴛʜᴇꜱᴇ ᴄᴏᴍᴍᴀɴᴅꜱ.__
```

Ruff (0.15.18)

[error] Invalid unescaped character backspace, use "\b" instead

Replace with escape sequence

(PLE2510)